cmd/podman/main_remote.go

```diff
@@ -20,11 +25,41 @@ func init() {
 	rootCmd.PersistentFlags().StringVar(&MainGlobalOpts.RemoteUserName, "username", username, "username on the remote host")
 	rootCmd.PersistentFlags().StringVar(&MainGlobalOpts.RemoteHost, "remote-host", "", "remote host")
 	// TODO maybe we allow the altering of this for bridge connections?
-	//rootCmd.PersistentFlags().StringVar(&MainGlobalOpts.VarlinkAddress, "varlink-address", adapter.DefaultAddress, "address of the varlink socket")
+	// rootCmd.PersistentFlags().StringVar(&MainGlobalOpts.VarlinkAddress, "varlink-address", adapter.DefaultAddress, "address of the varlink socket")
 	rootCmd.PersistentFlags().StringVar(&MainGlobalOpts.LogLevel, "log-level", "error", "Log messages above specified level: debug, info, warn, error, fatal or panic")
+	rootCmd.PersistentFlags().BoolVar(&MainGlobalOpts.Syslog, "syslog", false, "Output logging information to syslog as well as the console")
 }
 
 func setSyslog() error {
+	// Log to file if not using syslog
+	homeDir := homedir.Get()
+	path := filepath.Join(homeDir, ".config", "containers")
+
+	if _, err := os.Stat(path); os.IsNotExist(err) {
+		if err := os.MkdirAll(path, 0750); err != nil {
+			fmt.Fprintf(os.Stderr, "%v", err)
+			return err
+		}
+	}
+
+	// Update path to include file name
+	path = filepath.Join(path, "podman-remote.log")
```

Comment:
Can you just call this podman-remode.log.  When running podman on a Mac or Windows box, the command should be podman, not podman-remote.